### PR TITLE
Add email parameter to the manage account link

### PR DESCRIPTION
**Pull request by Holidaybear**

### CODE REVIEW

File: FirefoxPrivateNetworkVPN/ViewControllers/SettingsDataSource.swift [hunk 1]
```diff
@@ -92,7 +92,7 @@ extension SettingsDataSource: UITableViewDelegate {
 
         //swiftlint:disable:next trailing_closure
         headerView?.buttonTappedSubject.subscribe(onNext: { [weak self] _ in
-            self?.settingSelected.onNext(.account)
+            self?.settingSelected.onNext(.account(email: self?.account?.user.email))
```
Comment (pharspher) [inline FirefoxPrivateNetworkVPN/ViewControllers/SettingsDataSource.swift:95]: should this be self?.account?.user.email or self?.account?.user?.email

Comment (Holidaybear) [reply FirefoxPrivateNetworkVPN/ViewControllers/SettingsDataSource.swift:95]: `User` in `Account` won't be nil, so I think it's fine.

Comment (pharspher) [reply FirefoxPrivateNetworkVPN/ViewControllers/SettingsDataSource.swift:95]: Is it possible self or account here are nil?

Comment (Holidaybear) [reply FirefoxPrivateNetworkVPN/ViewControllers/SettingsDataSource.swift:95]: If self or account is nil, then the whole parameter would directly be nil, and we've skipped this inside when email parameter is nil.  👍

**Pull request merged**